tools/swarm/src/cli.rs

```diff
@@ -20,6 +20,11 @@ pub struct Cli {
     /// overwrite the file anyway, pass `--force` flag.
     #[arg(long, short)]
     pub outfile: PathBuf,
+    /// Put a banner in the file saying that the file is generated.
```

Comment:
The doc-comment should probably be updated to reflect the inverted logic of the flag